platform/mds/mds-web/src/main/resources/webapp/css/mds.css

```diff
@@ -462,4 +462,11 @@
 .mds .form-inline .form-group.inline,
 .mds .form-inline .form-group label {
     display: inline-block;
+}
+
+.ui-priority-secondary,
```

Comment:
We should move this to some global css file, so that this affects all datepickers, not only the mds ones.